### ISSUE TITLE
[Bug]: Fail to install the origin ControlNet

### ISSUE THREAD

**Issue by JoonSumisu** [lllyasviel/stable-diffusion-webui-forge#620]
### Checklist

- [X] The issue exists after disabling all extensions
- [X] The issue exists on a clean installation of webui
- [X] The issue is caused by an extension, but I believe it is caused by a bug in the webui
- [X] The issue exists in the current version of the webui
- [ ] The issue has not been reported before recently
- [ ] The issue has been reported before but has not been fixed yet

### What happened?

The builtin ControlNet does not function well with SDXL and unable to load the model, I tried to install the origin ControlNet webui in extension page.
`https://github.com/Mikubill/sd-webui-controlnet.git`

Firstly, after run the webui.sh, it shows 
`
Legacy Preprocessor init warning: Unable to install insightface automatically. Please try run `pip install insightface` manually.`

Secondly, in the extension page,  Apply and load won't activate the controlNet.

### Steps to reproduce the problem

1. Go to install from URL
2. Input the `https://github.com/Mikubill/sd-webui-controlnet.git`
3. Shows insightface install error
4. Unable to Apply and load the controlnet extension

### What should have happened?

Able to Apply and load the controlnet extension

### What browsers do you use to access the UI ?

Google Chrome

### Sysinfo

"Extensions": [],
    "Inactive extensions": [
        {
            "name": "sd-forge-layerdiffuse",
            "path": "/root/ai/stable-diffusion-webui-forge/extensions/sd-forge-layerdiffuse",
            "version": "e4d5060e",
            "branch": "main",
            "remote": "https://github.com/layerdiffusion/sd-forge-layerdiffuse.git"
        },
        {
            "name": "sd-webui-controlnet",
            "path": "/root/ai/stable-diffusion-webui-forge/extensions/sd-webui-controlnet",
            "version": "fcd542e5",
            "branch": "main",
            "remote": "https://github.com/Mikubill/sd-webui-controlnet.git"
        }
    ],
    "Environment": {
        "GIT": "git",
        "GRADIO_ANALYTICS_ENABLED": "False"
    },
    "Config": {
        "ldsr_steps": 100,
        "ldsr_cached": false,
        "SCUNET_tile": 256,
        "SCUNET_tile_overlap": 8,
        "SWIN_tile": 192,
        "SWIN_tile_overlap": 8,
        "SWIN_torch_compile": false,
        "control_net_detectedmap_dir": "detected_maps",
        "control_net_models_path": "",
        "control_net_modules_path": "",
        "control_net_unit_count": 3,
        "control_net_model_cache_size": 5,
        "control_net_no_detectmap": false,
        "control_net_detectmap_autosaving": false,
        "control_net_allow_script_control": false,
        "control_net_sync_field_args": true,
        "controlnet_show_batch_images_in_ui": false,
        "controlnet_increment_seed_during_batch": false,
        "controlnet_disable_openpose_edit": false,
        "controlnet_disable_photopea_edit": false,
        "controlnet_photopea_warning": true,
        "controlnet_input_thumbnail": true,
        "sd_checkpoint_hash": "8421598e930be75d6e7afc80db0ef38cf324901966bd0fb8739e69c448e18a89",
        "disabled_extensions": [
            "LDSR",
            "Lora",
            "ScuNET",
            "SwinIR",
            "canvas-zoom-and-pan",
            "extra-options-section",
            "forge_legacy_preprocessors",
            "forge_preprocessor_inpaint",
            "forge_preprocessor_marigold",
            "forge_preprocessor_normalbae",
            "forge_preprocessor_recolor",
            "forge_preprocessor_reference",
            "forge_preprocessor_revision",
            "forge_preprocessor_tile",
            "mobile",
            "prompt-bracket-checker",
            "sd_forge_dynamic_thresholding",
            "sd_forge_fooocus_inpaint",
            "sd_forge_freeu",
            "sd_forge_hypertile",
            "sd_forge_ipadapter",
            "sd_forge_kohya_hrfix",
            "sd_forge_latent_modifier",
            "sd_forge_multidiffusion",
            "sd_forge_neveroom",
            "sd_forge_photomaker",
            "sd_forge_sag",
            "sd_forge_stylealign",
            "sd_forge_svd",
            "sd_forge_z123",
            "soft-inpainting",
            "sd-forge-layerdiffuse"
        ],
        "disable_all_extensions": "none",
        "sd_model_checkpoint": "AnythingXL_xl.safetensors [8421598e93]"
    },
    "Startup": {
        "total": 4.472478866577148,
        "records": {
            "initial startup": 0.0039038658142089844,
            "prepare environment/checks": 4.458427429199219e-05,
            "prepare environment/git version info": 0.02696967124938965,
            "prepare environment/torch GPU test": 0.8382430076599121,
            "prepare environment/clone repositores": 0.004973649978637695,
            "prepare environment/run extensions installers": 0.0001533031463623047,
            "prepare environment/run extensions_builtin installers": 6.580352783203125e-05,
            "prepare environment": 0.8794510364532471,
            "launcher": 0.0004451274871826172,
            "import torch": 1.9675779342651367,
            "import gradio": 0.45052099227905273,
            "setup paths": 0.46221375465393066,
            "import ldm": 0.0035400390625,
            "import sgm": 3.5762786865234375e-06,
            "initialize shared": 0.025365591049194336,
            "other imports": 0.19893813133239746,
            "opts onchange": 0.00017690658569335938,
            "setup SD model": 2.86102294921875e-05,
            "setup codeformer": 0.00024056434631347656,
            "setup gfpgan": 0.0023407936096191406,
            "set samplers": 2.1457672119140625e-05,
            "list extensions": 0.0011036396026611328,
            "restore config state file": 7.3909759521484375e-06,
            "list SD models": 0.00015354156494140625,
            "list localizations": 2.4080276489257812e-05,
            "load scripts/custom_code.py": 0.0001773834228515625,
            "load scripts/img2imgalt.py": 0.0001392364501953125,
            "load scripts/loopback.py": 6.365776062011719e-05,
            "load scripts/outpainting_mk_2.py": 8.654594421386719e-05,
            "load scripts/poor_mans_outpainting.py": 5.1021575927734375e-05,
            "load scripts/postprocessing_caption.py": 3.719329833984375e-05,
            "load scripts/postprocessing_codeformer.py": 3.504753112792969e-05,
            "load scripts/postprocessing_create_flipped_copies.py": 3.314018249511719e-05,
            "load scripts/postprocessing_focal_crop.py": 0.00021457672119140625,
            "load scripts/postprocessing_gfpgan.py": 4.1484832763671875e-05,
            "load scripts/postprocessing_split_oversized.py": 4.100799560546875e-05,
            "load scripts/postprocessing_upscale.py": 7.2479248046875e-05,
            "load scripts/processing_autosized_crop.py": 4.696846008300781e-05,
            "load scripts/prompt_matrix.py": 5.3882598876953125e-05,
            "load scripts/prompts_from_file.py": 6.031990051269531e-05,
            "load scripts/sd_upscale.py": 4.57763671875e-05,
            "load scripts/xyz_grid.py": 0.0007746219635009766,
            "load scripts/comments.py": 0.10060667991638184,
            "load scripts/refiner.py": 0.00016188621520996094,
            "load scripts/seed.py": 0.00013566017150878906,
            "load scripts": 0.10290408134460449,
            "load upscalers": 0.0013997554779052734,
            "refresh VAE": 0.0007073879241943359,
            "refresh textual inversion templates": 3.62396240234375e-05,
            "scripts list_optimizers": 0.02255702018737793,
            "scripts list_unets": 7.62939453125e-06,
            "reload hypernetworks": 8.296966552734375e-05,
            "initialize extra networks": 0.0008080005645751953,
            "scripts before_ui_callback": 0.0002079010009765625,
            "create ui": 0.2991001605987549,
            "gradio launch": 0.05408215522766113,
            "add APIs": 0.0036346912384033203,
            "app_started_callback": 1.0013580322265625e-05
        }
    },
    "Packages": [
        "absl-py==2.1.0",
        "accelerate==0.21.0",
        "addict==2.4.0",
        "aenum==3.1.15",
        "aiofiles==23.2.1",
        "aiohttp==3.9.3",
        "aiosignal==1.3.1",
        "altair==5.2.0",
        "antlr4-python3-runtime==4.9.3",
        "anyio==3.7.1",
        "attrs==23.2.0",
        "basicsr==1.4.2",
        "blendmodes==2022",
        "certifi==2024.2.2",
        "cffi==1.16.0",
        "chardet==5.2.0",
        "charset-normalizer==3.3.2",
        "clean-fid==0.1.35",
        "click==8.1.7",
        "clip==1.0",
        "coloredlogs==15.0.1",
        "colorlog==6.8.2",
        "contourpy==1.2.0",
        "cssselect2==0.7.0",
        "cycler==0.12.1",
        "deprecation==2.1.0",
        "depth-anything==2024.1.22.0",
        "diffusers==0.25.0",
        "dsine==2024.3.23",
        "einops==0.4.1",
        "embreex==2.17.7.post4",
        "facexlib==0.3.0",
        "fastapi==0.94.0",
        "ffmpy==0.3.2",
        "filelock==3.13.3",
        "filterpy==1.4.5",
        "flatbuffers==24.3.25",
        "fonttools==4.50.0",
        "frozenlist==1.4.1",
        "fsspec==2024.3.1",
        "ftfy==6.2.0",
        "future==1.0.0",
        "fvcore==0.1.5.post20221221",
        "geffnet==1.0.2",
        "gitdb==4.0.11",
        "gitpython==3.1.32",
        "glob2==0.5",
        "gradio-client==0.5.0",
        "gradio==3.41.2",
        "grpcio==1.62.1",
        "h11==0.12.0",
        "handrefinerportable==2024.2.12.0",
        "httpcore==0.15.0",
        "httpx==0.24.1",
        "huggingface-hub==0.22.1",
        "humanfriendly==10.0",
        "idna==3.6",
        "imageio==2.34.0",
        "importlib-metadata==7.1.0",
        "importlib-resources==6.4.0",
        "inflection==0.5.1",
        "iopath==0.1.9",
        "jax==0.4.25",
        "jaxlib==0.4.25",
        "jinja2==3.1.3",
        "jsonmerge==1.8.0",
        "jsonschema-specifications==2023.12.1",
        "jsonschema==4.21.1",
        "kiwisolver==1.4.5",
        "kornia==0.6.7",
        "lark==1.1.2",
        "lazy-loader==0.3",
        "lightning-utilities==0.11.1",
        "llvmlite==0.42.0",
        "lmdb==1.4.1",
        "lxml==5.1.0",
        "mapbox-earcut==1.0.1",
        "markdown==3.6",
        "markupsafe==2.1.5",
        "matplotlib==3.8.3",
        "mediapipe==0.10.11",
        "ml-dtypes==0.3.2",
        "mpmath==1.3.0",
        "multidict==6.0.5",
        "networkx==3.2.1",
        "numba==0.59.1",
        "numpy==1.26.2",
        "omegaconf==2.2.3",
        "onnxruntime==1.17.1",
        "open-clip-torch==2.20.0",
        "opencv-contrib-python==4.9.0.80",
        "opencv-python==4.9.0.80",
        "opt-einsum==3.3.0",
        "orjson==3.9.15",
        "packaging==24.0",
        "pandas==2.2.1",
        "piexif==1.1.3",
        "pillow==9.5.0",
        "pip==23.2.1",
        "platformdirs==4.2.0",
        "portalocker==2.8.2",
        "protobuf==3.20.0",
        "psutil==5.9.5",
        "pycollada==0.8",
        "pycparser==2.21",
        "pydantic==1.10.14",
        "pydub==0.25.1",
        "pyparsing==3.1.2",
        "python-dateutil==2.9.0.post0",
        "python-multipart==0.0.9",
        "pytorch-lightning==1.9.4",
        "pytz==2024.1",
        "pywavelets==1.5.0",
        "pyyaml==6.0.1",
        "referencing==0.34.0",
        "regex==2023.12.25",
        "reportlab==4.1.0",
        "requests==2.31.0",
        "resize-right==0.0.2",
        "rpds-py==0.18.0",
        "rtree==1.2.0",
        "safetensors==0.4.2",
        "scikit-image==0.21.0",
        "scipy==1.12.0",
        "semantic-version==2.10.0",
        "sentencepiece==0.2.0",
        "setuptools==65.5.0",
        "shapely==2.0.3",
        "six==1.16.0",
        "smmap==5.0.1",
        "sniffio==1.3.1",
        "sounddevice==0.4.6",
        "spandrel==0.1.6",
        "starlette==0.26.1",
        "svg.path==6.3",
        "svglib==1.5.1",
        "sympy==1.12",
        "tabulate==0.9.0",
        "tb-nightly==2.17.0a20240326",
        "tensorboard-data-server==0.7.2",
        "termcolor==2.4.0",
        "tifffile==2024.2.12",
        "timm==0.9.16",
        "tinycss2==1.2.1",
        "tokenizers==0.13.3",
        "tomesd==0.1.3",
        "tomli==2.0.1",
        "toolz==0.12.1",
        "torch==2.1.2+cu121",
        "torchdiffeq==0.2.3",
        "torchmetrics==1.3.2",
        "torchsde==0.2.6",
        "torchvision==0.16.2+cu121",
        "tqdm==4.66.2",
        "trampoline==0.1.2",
        "transformers==4.30.2",
        "trimesh==4.2.2",
        "triton==2.1.0",
        "typing-extensions==4.10.0",
        "tzdata==2024.1",
        "urllib3==2.2.1",
        "uvicorn==0.29.0",
        "vhacdx==0.0.6",
        "wcwidth==0.2.13",
        "webencodings==0.5.1",
        "websockets==11.0.3",
        "werkzeug==3.0.1",
        "xatlas==0.0.9",
        "xxhash==3.4.1",
        "yacs==0.1.8",
        "yapf==0.40.2",
        "yarl==1.9.4",
        "zipp==3.18.1"
    ]
}

### Console logs

```Shell
################################################################
Install script for stable-diffusion + Web UI
Tested on Debian 11 (Bullseye), Fedora 34+ and openSUSE Leap 15.4 or newer.
################################################################

################################################################
Running on root user
################################################################

################################################################
Repo already cloned, using it as install directory
################################################################

################################################################
Create and activate python venv
################################################################

################################################################
Launching launch.py...
################################################################
glibc version is 2.35
Cannot locate TCMalloc. Do you have tcmalloc or google-perftool installed on your system? (improves CPU memory usage)
Python 3.11.5 (main, Sep 11 2023, 13:54:46) [GCC 11.2.0]
Version: f0.0.17v1.8.0rc-latest-276-g29be1da7
Commit hash: 29be1da7cf2b5dccfc70fbdd33eb35c56a31ffb7
Launching Web UI with arguments: -f
Total VRAM 24576 MB, total RAM 32022 MB
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 3090 : native
Hint: your device supports --pin-shared-memory for potential speed improvements.
Hint: your device supports --cuda-malloc for potential speed improvements.
Hint: your device supports --cuda-stream for potential speed improvements.
VAE dtype: torch.bfloat16
CUDA Stream Activated:  False
Using pytorch cross attention
Loading weights [8421598e93] from /root/ai/stable-diffusion-webui-forge/models/Stable-diffusion/AnythingXL_xl.safetensors
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
gio: http://127.0.0.1:7860/: Operation not supported
Startup time: 4.6s (prepare environment: 0.9s, import torch: 1.9s, import gradio: 0.4s, setup paths: 0.5s, other imports: 0.2s, create ui: 0.3s, gradio launch: 0.3s).
model_type EPS
UNet ADM Dimension 2816
Using pytorch attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using pytorch attention in VAE
extra {'cond_stage_model.clip_l.logit_scale', 'cond_stage_model.clip_l.text_projection', 'cond_stage_model.clip_g.transformer.text_model.embeddings.position_ids'}
To load target model SDXLClipModel
Begin to load 1 model
[Memory Management] Current Free GPU Memory (MB) =  23305.99609375
[Memory Management] Model Memory (MB) =  2144.3546981811523
[Memory Management] Minimal Inference Memory (MB) =  1024.0
[Memory Management] Estimated Remaining GPU Memory (MB) =  20137.641395568848
Moving model(s) has taken 0.27 seconds
Model loaded in 2.3s (load weights from disk: 0.5s, forge load real models: 1.4s, calculate empty prompt: 0.4s).

################################################################
Launching launch.py...
################################################################
glibc version is 2.35
Cannot locate TCMalloc. Do you have tcmalloc or google-perftool installed on your system? (improves CPU memory usage)
Python 3.11.5 (main, Sep 11 2023, 13:54:46) [GCC 11.2.0]
Version: f0.0.17v1.8.0rc-latest-276-g29be1da7
Commit hash: 29be1da7cf2b5dccfc70fbdd33eb35c56a31ffb7
Launching Web UI with arguments: -f
Total VRAM 24576 MB, total RAM 32022 MB
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 3090 : native
Hint: your device supports --pin-shared-memory for potential speed improvements.
Hint: your device supports --cuda-malloc for potential speed improvements.
Hint: your device supports --cuda-stream for potential speed improvements.
VAE dtype: torch.bfloat16
CUDA Stream Activated:  False
Using pytorch cross attention
Loading weights [8421598e93] from /root/ai/stable-diffusion-webui-forge/models/Stable-diffusion/AnythingXL_xl.safetensors
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Startup time: 4.5s (prepare environment: 0.9s, import torch: 2.0s, import gradio: 0.5s, setup paths: 0.5s, other imports: 0.2s, load scripts: 0.1s, create ui: 0.3s).
model_type EPS
UNet ADM Dimension 2816
Using pytorch attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using pytorch attention in VAE
extra {'cond_stage_model.clip_g.transformer.text_model.embeddings.position_ids', 'cond_stage_model.clip_l.text_projection', 'cond_stage_model.clip_l.logit_scale'}
To load target model SDXLClipModel
Begin to load 1 model
[Memory Management] Current Free GPU Memory (MB) =  23305.99609375
[Memory Management] Model Memory (MB) =  2144.3546981811523
[Memory Management] Minimal Inference Memory (MB) =  1024.0
[Memory Management] Estimated Remaining GPU Memory (MB) =  20137.641395568848
Moving model(s) has taken 0.34 seconds
Model loaded in 2.5s (load weights from disk: 0.4s, forge load real models: 1.5s, calculate empty prompt: 0.5s).
```
